src/pages/docs/run-anywhere/adding-provisioners-and-admins.mdx

```diff
@@ -44,22 +44,22 @@ To review the configuration for all existing provisioners on your certificate au
 step ca provisioner list
 ```
 
-Check the existing configuration for a given provisioner by running [`step beta ca provisioner get`](https://smallstep.com/docs/step-cli/reference/beta/ca/provisioner/get):
+Check the existing configuration for a given provisioner by running [`step ca provisioner get`](https://smallstep.com/docs/step-cli/reference/beta/ca/provisioner/get):
 
 ```bash
-step beta ca provisioner get <provisioner-name>
+step ca provisioner get <provisioner-name>
 ```
 
-To add a new provisioner, use [`step beta ca provisioner add`](https://smallstep.com/docs/step-cli/reference/beta/ca/provisioner/add). There are several options for find-tuning a provisioner for your use case. On all provisioner types, you can constrain certificate lifetimes, configure templates to be used, and specify provisioner admins. Provisioner admins can issue certificates with arbitrary subjects. The [command reference](https://smallstep.com/docs/step-cli/reference/beta/ca/provisioner/add) describes other provisioner-specific configuration options that are available.
+To add a new provisioner, use [`step ca provisioner add`](https://smallstep.com/docs/step-cli/reference/beta/ca/provisioner/add). There are several options for find-tuning a provisioner for your use case. On all provisioner types, you can constrain certificate lifetimes, configure templates to be used, and specify provisioner admins. Provisioner admins can issue certificates with arbitrary subjects. The [command reference](https://smallstep.com/docs/step-cli/reference/beta/ca/provisioner/add) describes other provisioner-specific configuration options that are available.
 
 If you just want stand up a basic, password-based provisioner with default values to get moving, run:
 
 ```bash
-step beta ca provisioner add <provisioner-name> --type JWK --create
+step ca provisioner add <provisioner-name> --type JWK --create
 ```
 
 You will be prompted to enter a password for the provisioner.
 
 > For more information on each provisioner type and recommended configurations, please see our open source documentation on [How to Configure Provisioners](https://smallstep.com/docs/step-ca/provisioners#authorization-scope-by-provisioner).
-> Our open source documentation was written for `step-ca`, so it instructs you to use the `step ca` commands instead of the `step beta` commands. Instead, use the `step beta` commands listed in the guide above.
+> Our open source documentation was written for `step-ca`, so it instructs you to use the `step ca` commands instead of the `step` commands. Instead, use the `step` commands listed in the guide above.
```

Comment:
What does this line of documentation mean? Can it be removed?

Reply:
Yes I thin this was referring to `step beta` commands as something separate. good catch.